### PR TITLE
Update harbour-seaprint-es.ts

### PR DESCRIPTION
**Pull request by JSEHV**
Update Spanish translation with latest strings

### CODE REVIEW

**Comment by attah** [comment]
Thanks! I'll review this when i have some more time/energy

**Comment by attah** [comment]
Quick initial feedback... Qt  complains that &eacute; is not a thing.. so presumably any things (acutes, tilde, ..) that were not escaped before doesn't need to be.

**Comment by JSEHV** [comment]
Should be all fixed now in the second commit to this pull request.
Hope you can merge it with version 1.0.4 :)

File: translations/harbour-seaprint-es.ts [hunk 1]
```diff
@@ -402,90 +402,90 @@
     </message>
     <message>
         <source>Page range</source>
-        <translation type="unfinished"></translation>
+        <translation>Fuente de medios</translation>
```
Comment (attah) [inline translations/harbour-seaprint-es.ts]: This doesn't look correct - seems duplicated from "media source"

File: translations/harbour-seaprint-es.ts [hunk 2]
```diff
@@ -538,45 +538,45 @@
...
     </message>
     <message>
         <source>Ignore SSL errors</source>
-        <translation type="unfinished"></translation>
+        <translation>Ignorar errores SSL</translation>
     </message>
     <message>
         <source>In order to work with self-signed certificates of printers and CUPS instances, SSL errors needs to be ignored.</source>
-        <translation type="unfinished"></translation>
+        <translation>Para trabajar con certificados autofirmados de impresoras e instancias CUPS, es necesario ignorar los errores SSL.</translation>
```
Comment (attah) [inline translations/harbour-seaprint-es.ts]: Is this the correct type of "work"?

File: translations/harbour-seaprint-es.ts [hunk 3]
```diff
@@ -783,115 +783,115 @@
...
     </message>
     <message>
         <source>single document</source>
-        <translation type="unfinished"></translation>
+        <translation>documento único</translation>
     </message>
     <message>
         <source>uncollated</source>
-        <translation type="unfinished"></translation>
+        <translation>inacabado</translation>
```
Comment (attah) [inline translations/harbour-seaprint-es.ts]: should uncollated/collated not be versions of the same word?

**Review by attah — COMMENTED** [review]
Looks pretty good - some minor comments.
But my Spanish is extremely rusty...
Even Qt creator makes zero adjustments when building. :)

**Comment by attah** [comment]
Thank you for your contribution!

**Pull request merged**